remote/inc/block-patterns.php

```diff
@@ -14,7 +14,7 @@
  */
 function remote_register_block_patterns() {
 	$block_pattern_categories = array(
-		'pages' => array( 'label' => __( 'Pages', 'remote' ) ),
+		'remote' => array( 'label' => __( 'Remote', 'remote' ) ),
```

Comment:
This will be affected by https://github.com/Automattic/themes/pull/5664
In this particular case, I think the categories we want for the quote pattern are text and featured. We are not adding a theme specific category

Reply:
Cool. Let me revert this change.